mainwindow.cc

```diff
@@ -584,16 +584,19 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
 
   connect( ui.menuHistory, SIGNAL( aboutToShow() ),
            this, SLOT( updateHistoryMenu() ) );
-
+#if !defined( HAVE_X11 ) || QT_VERSION < QT_VERSION_CHECK( 5, 0, 0 )
```

Comment:
Please be more careful. Restore the empty line, which separated the unrelated chunks of code.

Reply:
Done